lang/resources/org/partiql/type-domains/partiql.ion

```diff
@@ -518,30 +518,6 @@ may then be further optimized by selecting better implementations of each operat
             // Special types
             (any_type)
 
-            // TODO: Remove these hardcoded nodes from the PIG domain once [https://github.com/partiql/partiql-lang-kotlin/issues/510]
```

Comment:
There are still some references to #510 in AstSerialization.kt. Can those references also be removed?

- https://github.com/partiql/partiql-lang-kotlin/blob/342c2d23e933d032e278d4df26891aed78507eb8/lang/src/org/partiql/lang/ast/AstDeserialization.kt#L183-L184
- https://github.com/partiql/partiql-lang-kotlin/blob/342c2d23e933d032e278d4df26891aed78507eb8/lang/src/org/partiql/lang/ast/AstDeserialization.kt#L261-L262
- https://github.com/partiql/partiql-lang-kotlin/blob/342c2d23e933d032e278d4df26891aed78507eb8/lang/src/org/partiql/lang/ast/AstDeserialization.kt#L291-L292

Reply:
I think these references present a generic custom type node (the same as https://github.com/partiql/partiql-lang-kotlin/blob/main/lang/resources/org/partiql/type-domains/partiql.ion#L545), instead of a specific custom type node, such as `rs_varchar`. So I don't think we are able to remove them

Reply:
Agreeing with @lziq that any objection above is now more than mute, after AstDeserialization.kt is now completely gone, in #863.